src/styles/blocks/blocks.scss

```
@@ -0,0 +1,30 @@
+.blocks {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: #fff;
+
+  @media (min-width: 1050px) {
+    grid-template-columns: repeat(3, minmax(300px, 1fr));
+  }

```

```suggestion
  @media (min-width: 1200px) {
    grid-template-columns: repeat(3, 1fr);
  }

```
3 blocks * 300px = 1200. Why 1050px?

---

Согласен, натупил, почему-то решил что нужны еще отступы.